src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { Loader } from '../components/Loader';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isPeopleLoading, setIsPeopleLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  const { personId } = useParams();
+  const selectedPersonId = personId;
+
+  function getPeopleWithParents(peopleResponce: Person[]) {
```

Comment:
There is a typo in the function name `getPeopleWithParents`. It should be `getPeopleWithParents` instead of `getPeopleWithParents`.